client/pom.xml

```diff
@@ -179,22 +195,27 @@
           <plugin>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-codegen-plugin</artifactId>
+            <version>${cxf.version}</version>
             <executions>
               <execution>
-                <id>import-wsdl</id>
+                <id>generate-sources</id>
                 <phase>generate-sources</phase>
                 <goals>
                   <goal>wsdl2java</goal>
                 </goals>
                 <configuration>
+                  <sourceRoot>${project.build.directory}/generated-sources/cxf</sourceRoot>
                   <wsdlOptions>
-                      <wsdlOption>
-                          <wsdl>${project.build.directory}/winrm4j-service-wsdl/META-INF/wsdl/WinRmService.wsdl</wsdl>
-                          <bindingFiles><bindingFile>${basedir}/src/jaxws/bindings.xml</bindingFile></bindingFiles>
-                          <extraargs>
-                              <extraarg>-verbose</extraarg>
-                          </extraargs>
-                      </wsdlOption>
+                    <wsdlOption>
+                      <!--Use this wsdl file to generate classes directly from there-->
+                      <!--<wsdl>${basedir}/src/main/resources/ref_wsdl/WinRmService.wsdl</wsdl>-->
+                      <wsdl>${basedir}/../service/target/classes/wsdl/WinRmService.wsdl</wsdl>
```

Comment:
Should use the file which the `maven-dependency-plugin` plugin has extracted above. Don't rely on the service being there (will be fetched from a maven repo if missing).